core/trino-main/src/main/java/io/trino/execution/scheduler/PipelinedQueryScheduler.java

```diff
@@ -90,7 +90,7 @@
 import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
Can we test this change somehow?

Reply:
Not so easy, unfortunately. I would rather not test this change. This change is relatively simple.